test/images/scripts/fedora-26.setup

```diff
@@ -70,6 +70,15 @@ gdb \
 targetcli \
 "
 
+# For correct work of ABRT in Fedora 26 Alpha release a following packages
```

Comment:
Lets mark this with ```HACK:``` and a link to the bugzilla URL that solves the issue long term.